source/unified-test-format/unified-test-format.rst

```diff
@@ -582,6 +583,58 @@ The structure of this object is as follows:
 
   - ``serverApi``: Optional `serverApi`_ object.
 
+.. _entity_encryptedClient:
+
+- ``encryptedClient``: Optional object. Defines a ClientEncryption object.
+
+  The structure of this object is as follows:
+
+  - ``id``: Required string. Unique name for this entity. The YAML file SHOULD
+    define a `node anchor`_ for this field (e.g.
+    ``id: &encryptedClient0 encryptedClient0``).
```

Comment:
I'd propose adding a `client` option here, since that tell us what client to use for constructing the ClientEncryption object. And then `keyVaultClient` can default to this if left unspecified, which I think is consistent with the current driver API for creating ClientEncryption objects.

Let me know if I'm mistaken and this is changing in the current CSFLE work.